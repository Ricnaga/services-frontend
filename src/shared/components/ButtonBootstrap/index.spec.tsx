import { render } from '@testing-library/react';
import { ButtonBootstrap } from '.';

describe('BUTTON COMPONENT', () => {
  it('should be able to render component', () => {
    const { container } = render(<ButtonBootstrap title="ButtonBootstrap" />);

    expect(container).toMatchSnapshot();
  });
});
