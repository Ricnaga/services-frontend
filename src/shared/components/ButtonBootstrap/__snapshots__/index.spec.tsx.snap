// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BUTTON COMPONENT should be able to render component 1`] = `
<div>
  <button
    class="btn btn-outline-warning btn-lg"
    type="submit"
  >
    ButtonBootstrap
  </button>
</div>
`;
